### DISCUSSION TITLE
Can’t connect Github to Netlify to start existing project. Error: Page is taking too long to load

### DISCUSSION THREAD

**Discussion by justsilencia** [community/community#36063] [Other Feature Feedback, Questions, & Ideas]
I already have 3 github repos linked to my netlify account. For some reason, when I try to add another repo by clicking the Configure the Netlify app on Github in the image below, I keep getting the error with the unicorn image saying “This page is taking too long to load.” This has been happening since Monday. Please help; much appreciated. Thanks.

Please see the below images:

I click the Configure the Netlify app on Github button at the bottom:

![2](https://user-images.githubusercontent.com/48294438/195676073-3ca4c2b9-8fff-4fb0-a1f7-dd04b7f1f66d.PNG)

And I get the following error:

![3](https://user-images.githubusercontent.com/48294438/195676151-47e676ad-6cb0-4d02-a3ad-7e66a206cf10.PNG)

**Comment by mr-palindrome** [comment]
I think I found the solution. Try leaving those organisation where you are collaborator!
It worked for me!

**Reply by Hilljackcantspell** [reply]
> I think I found the solution. Try leaving those organisation where you are collaborator! It worked for me!

**Reply by brenio55** [reply]
What do you mean? I didn't get it
